extensions/wikia/Recirculation/RecirculationController.class.php

```diff
@@ -58,9 +58,30 @@ public function footer() {
 		$this->response->setVal( 'communityHeaderBackground',
 			$themeSettings->getCommunityHeaderBackgroundUrl() );
 		$this->response->setVal( 'sitename', $wgSitename );
+		$this->response->setVal( 'topWikiArticles', $this->getTopWikiArticles() );
+		// TODO if topWikiArticles is empty show one more N&S article card instead of More from wiki card
 		$this->response->setTemplateEngine( WikiaResponse::TEMPLATE_ENGINE_PHP );
 	}
 
+	private function getTopWikiArticles() {
+		global $wgCityId;
+
+		$topWikiArticles = TopWikiArticles::getArticlesWithCache( $wgCityId, false );
+		// do not show itself
+		$topWikiArticles = array_filter( $topWikiArticles, function ( $item ) {
+			return $item['id'] !== RequestContext::getMain()->getTitle()->getArticleID();
+		} );
+		// show max 3 elements
+		$topWikiArticles = array_slice( $topWikiArticles, 0, 3 );
+		// add index to items to render it by mustache template
+		$index = 1;
+		foreach ( $topWikiArticles as &$wikiArticle ) {
+			$wikiArticle['index'] = $index++;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Extract this increment or decrement operator into a dedicated statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS881)